src/page/address-step.page.ts

```diff
@@ -0,0 +1,13 @@
+import { $, ElementFinder } from 'protractor';
+
+export class AddressStepPage {
+  private element: ElementFinder;
+
+  constructor () {
+    this.element = $('#center_column > form > p > button > span');
+  }
+
+  public async addressStep(): Promise<void> {
+    await this.element.click();
+  }
```

Comment:
remueve async await y retorna ` this.element.click();`

Reply:
no lo estamos manejando así German :p